src/Workspaces/Core/Portable/Shared/Extensions/INamedTypeSymbolExtensions.cs

```diff
@@ -361,8 +362,11 @@ private static bool IsAbstractPropertyImplemented(INamedTypeSymbol classOrStruct
                 foreach (var member in type.GetMembers())
                 {
                     var namedParameter = IsAttributeNamedParameter(member, within ?? compilation.Assembly);
-                    if (namedParameter != null)
+                    if (namedParameter != null && !suggestedMembers.Contains(namedParameter.Name))
```

Comment:
A simpler and more efficient fix would be to just ignore symbols whose [IsOverride](http://source.roslyn.io/#Microsoft.CodeAnalysis/Symbols/ISymbol.cs,100) property is true.

Reply:
I realized that my proposed fix will not handle shadowing members, and there is no property on `ISymbol` that gives this information.

I think the correct fix here is either to perform the symbol filtering at callsites of this API or have this API take a special bool parameter that makes the filtering behavior explicit, so future callsites don't need to look at the implementation to correctly understand the API semantics.